src/plots/gl3d/scene.js

```diff
@@ -172,6 +172,15 @@ function initializeGLPlot(scene, fullLayout, canvas, gl) {
         showNoWebGlMsg(scene);
     }
 
+    var relayoutCallback = function(scene, domEvent) {
+        var update = {};
+        update[scene.id] = getLayoutCamera(scene.camera);
+        scene.graphDiv.emit('plotly_relayout', update);
+    };
+
+    scene.glplot.canvas.addEventListener('mouseup', relayoutCallback.bind(null, scene));
+    scene.glplot.canvas.addEventListener('wheel', relayoutCallback.bind(null, scene));
```

Comment:
Great. It would be nice to add a mouse wheel test. See the [legend_scroll](https://github.com/plotly/plotly.js/blob/30b9cc62a6ba716a3f772ce1eb3b8b30bf8367b4/test/jasmine/tests/legend_scroll_test.js#L185-L187) for inspiration.